NINJ2.py

```diff
@@ -0,0 +1,645 @@
+import os
+from Bio import SeqIO
+from Bio.SeqUtils import GC
+
+
+def fastq_filtrator(input_path: str, output_filename=None,
+                    gc_bounds=None, length_bounds=(0, 2 ** 32), quality_threshold=0):
```

Comment:
Интервал по умолчанию для ГЦ-состава (20, 80).
И тут же можно анотаацию типов добавить
```suggestion
                    gc_bounds: int | tuple = (20, 80), 
                    length_bounds: int | tuple = (0, 2 ** 32), 
                    quality_threshold: int = 0):
```